spec/controllers/exchange_rates_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe ExchangeRatesController, type: :controller do
+  before { TradeTariffFrontend::ServiceChooser.service_choice = nil }
+
+  context 'with xi as the service choice' do
+    before do
+      allow(TradeTariffFrontend::ServiceChooser).to receive(:xi?).and_return(true)
+      TradeTariffFrontend::ServiceChooser.service_choice = 'xi'
+    end
+
+    context 'when accessing the index action' do
```

Comment:
I think we should use describe here,
something like:
`describe('GET #index')`

Reply:
using routes now so no need for specs